### PR TITLE
Upgrade deps.

### PR DESCRIPTION
**Pull request by jgerigmeyer**

### CODE REVIEW

File: src/scss/config/_scale.scss [hunk 1]
```diff
@@ -75,7 +75,7 @@ $spacing-sizes: (
 /// @group config-scale
 $theme-sizes: (
   'page-margin': calc(var(--quarter-shim) + 4vw),
-  'border-width': 3px,
+  'border-width': medium,
```
Comment (jgerigmeyer) [inline src/scss/config/_scale.scss:78]: @stacyk This results in the updated HTML as expected, but the border itself is not visible.

Screen Shot 2022-08-22 at 12 28 03 PM

Comment (jgerigmeyer) [reply src/scss/config/_scale.scss:78]: @stacyk Looks good!

Screen Shot 2022-08-24 at 1 28 43 PM

**Review by jgerigmeyer — COMMENTED** [review]
@stacyk Looks like I need approval here before merging...

**Review by stacyk — APPROVED** [review]
yay

**Pull request merged**